DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Features/GeometryToSpeckleBase.cs

```diff
@@ -1,71 +1,58 @@
 using ArcGIS.Core.Geometry;
-using Speckle.Autofac.DependencyInjection;
 using Speckle.Converters.Common;
 using Speckle.Converters.Common.Objects;
 using Speckle.Core.Models;
 
 namespace Speckle.Converters.ArcGIS3.Features;
 
-[NameAndRankValue(nameof(ArcGIS.Core.Geometry.Geometry), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
-public class GeometryToSpeckleBaseList
-  : IHostObjectToSpeckleConversion,
-    IRawConversion<ArcGIS.Core.Geometry.Geometry, Base>
+public class GeometryToSpeckleBaseList : IRawConversion<ArcGIS.Core.Geometry.Geometry, List<Base>>
 {
-  private readonly IFactory<string, IHostObjectToSpeckleConversion> _toSpeckle;
-  private readonly IRawConversion<MapPoint, Base> _pointFeatureConverter;
-  private readonly IRawConversion<Multipoint, Base> _multiPointFeatureConverter;
-  private readonly IRawConversion<Polyline, Base> _polylineFeatureConverter;
-  private readonly IRawConversion<Polygon, Base> _polygonFeatureConverter;
+  private readonly IRawConversion<MapPoint, List<Base>> _pointFeatureConverter;
+  private readonly IRawConversion<Multipoint, List<Base>> _multiPointFeatureConverter;
+  private readonly IRawConversion<Polyline, List<Base>> _polylineFeatureConverter;
+  private readonly IRawConversion<Polygon, List<Base>> _polygonFeatureConverter;
 
   public GeometryToSpeckleBaseList(
-    IFactory<string, IHostObjectToSpeckleConversion> toSpeckle,
-    IRawConversion<MapPoint, Base> pointFeatureConverter,
-    IRawConversion<Multipoint, Base> multiPointFeatureConverter,
-    IRawConversion<Polyline, Base> polylineFeatureConverter,
-    IRawConversion<Polygon, Base> polygonFeatureConverter
+    IRawConversion<MapPoint, List<Base>> pointFeatureConverter,
+    IRawConversion<Multipoint, List<Base>> multiPointFeatureConverter,
+    IRawConversion<Polyline, List<Base>> polylineFeatureConverter,
+    IRawConversion<Polygon, List<Base>> polygonFeatureConverter
   )
   {
-    _toSpeckle = toSpeckle;
     _pointFeatureConverter = pointFeatureConverter;
     _multiPointFeatureConverter = multiPointFeatureConverter;
     _polylineFeatureConverter = polylineFeatureConverter;
     _polygonFeatureConverter = polygonFeatureConverter;
   }
 
-  public Base Convert(object target) => RawConvert((ArcGIS.Core.Geometry.Geometry)target);
-
-  public Base RawConvert(ArcGIS.Core.Geometry.Geometry target)
+  public List<Base> RawConvert(ArcGIS.Core.Geometry.Geometry target)
   {
-    List<Base> convertedList = new();
+    List<Base> convertedList;
 
-    Type type = target.GetType();
     try
     {
-      Base newGeometry = new(); // objectConverter.Convert(target);
-      if (target is MapPoint)
+      if (target is MapPoint point)
       {
-        newGeometry = _pointFeatureConverter.RawConvert((MapPoint)target);
-        convertedList.Add(newGeometry);
-        return convertedList[0];
+        convertedList = _pointFeatureConverter.RawConvert(point);
```

Comment:
This is the conversion I think is redundant. Just injecting the `IRawConversion<MapPoint,SOG.Point>` and wrapping it's result in a list here should be enough.

Reply:
same as above